index.html

```diff
@@ -1030,10 +1030,16 @@
 which is just a set of characteristics that is too small or unstable to be useful
 for following them through time.
 
+It is important to keep in mind that a person's [=identity=] may differ from their
+[=legal identity=]. A [=person=]'s <dfn>legal identity</dfn> is the set of characteristics
+that a governmental or otherwise official entity recognizes as corresponding to that
+[=person=].
```

Comment:
Maybe:
```suggestion
that a governmental or otherwise official entity uses to identify/recognize that
[=person=].
```

Or pick just one of identify or recognize.